MyString.java

```diff
@@ -18,13 +18,40 @@ public static void main(String[] args) {
 
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
-        // Replace the following statement with your code
-        return null;
+        String lowerCase = "";
+
+        for (int i = 0; i < str.length(); i++) {
+            char ch = str.charAt(i);
+            if (ch >= 65 && ch <= 90) {
+                ch += 32;
+                lowerCase += ch;
+            }
+            else {
+                lowerCase += ch;
+            }
+        }
+        return lowerCase;
     }
 
     /** If str1 contains str2, returns true; otherwise returns false. */
     public static boolean contains(String str1, String str2) {
-        // Replace the following statement with your code
+        int lenght1 = str1.length();
+        int length2 = str2.length();
+
+        if (length2 > lenght1) {
+            return false;
+        }
+        for (int i = 0; i < lenght1; i++) {
+            if (i + length2 > lenght1) {
+                break;
+            }
+            String subString1 = str1.substring(i, i + length2);
+            for (int j = 0; j < lenght1; j++) {
+                if (str2.indexOf(subString1) != -1) {
+                    return true;
```

Comment:
1. The second loop repeat the same check over and over again.. 
2. According to rules of HW, `string.indexOf(String str)` wasn't taught in lecture or recitation, and basically solves the question, since hypothetically. 
`return str1.indexOf(str2) != -1; `
is a correct solution..
However, but on looking at this is a very close case, 
After careful consideration, I have decided that I am not going to mark down your grade, 
For the following reasons:
A. It does look like you solved in the indented way
B. the problematic line(50) could be replaced with `if(str2.equals(subString1))`.  which is not problematic..